GETTING_STARTED.md

```diff
@@ -39,10 +39,20 @@ Now that you've set up your local environment and chosen a demo, you can deploy
 ```console
 $ docker run -it\
     -v $HOME/configs/:/mnt/deployer/configs/\
-    --entrypoint ruby ghcr.io/newrelic/deployer main.rb -c configs/<username>.docker.local.json -d <demo-url>
+    --entrypoint ruby ghcr.io/newrelic/deployer main.rb -d <demo-url>
+```
+
+Don't forget to replace `<demo-url>` with the url or local path to a deployment configuration file.
+
+If you named your user config file something other than 'user.credentials.json' use this command:
```

Comment:
We have already pointed the '-c' flag out above, maybe we could leave this all out and keep it more simple